scripts/network-json-validator

```diff
@@ -100,7 +100,7 @@ class CrowbarConduit
 
     @if_list.each do |iface|
       m = /^([-+?]?)(\d{1,3}[mg])(\d+)$/.match(iface) # [1]=sign, [2]=speed, [3]=count
-      if m.nil? || !%w{10m 100m 1g 10g}.include?(m[2])
+      if m.nil? || !%w{10m 100m 1g 10g 40g 56g}.include?(m[2])
```

Comment:
AFAIK our conduit resolver (and the ohai plugin) "knows" about 20g interface as well. Even if these are probably now widely used, we might want to bring this in sync with:

https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/lib/crowbar/conduit_resolver.rb#L154

Reply:
Done, thanks!